src-electron/db/query-impexp.js

```diff
@@ -626,7 +626,8 @@ async function importAttributeForEndpointType(
   packageIds,
   endpointTypeId,
   endpointClusterId,
-  attribute
+  attribute,
+  cluster
```

Comment:
Update function documentation with the new parameter

Reply:
oh, good catch, thanks